src/messages/manifestjson.js

```diff
@@ -0,0 +1,9 @@
+import { gettext as _ } from 'utils';
+
+export const MANIFEST_VERSION_INVALID = {
+  code: 'MANIFEST_VERSION_INVALID',
+  legacyCode: null,
+  message: _('"manifest_version" in the manifest.json is not a valid value'),
+  description: _('See https://goo.gl/WyioRC (MDN Docs) for more information.'),
```

Comment:
I don’t, and as discussed in IRC it didn’t seem easy to find. I say ship this, it’s cool.